stdlib/Profile/src/Profile.jl

```diff
@@ -1323,7 +1327,7 @@ function _stream_heap_snapshot(prefix::AbstractString, all_one::Bool)
         end
     end
 end
-function take_heap_snapshot(all_one::Bool=false; dir::Union{Nothing,S}=nothing) where {S <: AbstractString}
+function take_heap_snapshot(all_one::Bool=false; redact_data::Bool=true, dir::Union{Nothing,S}=nothing, streaming::Bool=false) where {S <: AbstractString}
```

Comment:
I think we'd end up with less duplication if we leave off the unused kwargs here:
```suggestion
function take_heap_snapshot(all_one::Bool=false; dir::Union{Nothing,S}=nothing, kwargs...) where {S <: AbstractString}
```